test/contrib/http/miniapp_test.rb

```diff
@@ -10,16 +10,17 @@
 # as ES happens to A) serve http and B) be supported with
 # a dedicated integration, so it's already there in our CI.
 class HTTPMiniAppTest < Minitest::Test
-  ELASTICSEARCH_HOST = '127.0.0.1'.freeze
-  ELASTICSEARCH_PORT = 49200
+  ELASTICSEARCH_HOST = ENV.fetch('TEST_ELASTICSEARCH_HOST', '127.0.0.1').freeze
+  ELASTICSEARCH_PORT = ENV.fetch('TEST_ELASTICSEARCH_PORT', '9200').freeze
```

Comment:
So this integer/string thing is a little annoying because the adapter needs an integer, but the tag comparison needs a string, so someplace or another we're going to end up doing an integer or string conversion. Maybe this should be an integer by default though.